internal/infrastructure/logger/init.go

```diff
@@ -48,6 +48,16 @@ func InitLogger(cfg *config.Config) {
 		panic("'cfg' cannot be nil")
 	}
 
+	// This will be always logged on service startup
+	slog.Info(
+		"idmsvc-backend",
+		slog.Group("Logging",
+			slog.Bool("Console", cfg.Logging.Console),
+			slog.String("Type", cfg.Logging.Type),
+			slog.String("Level", cfg.Logging.Level),
+		),
+	)
+
```

Comment:
I guess we would need to move the block at the end of the function, so the metahandler and cloudwatch are set as the default slog, and the level of the log.

Reply:
my bad, I understood in a different way :+1: